src/test/java/edu/harvard/iq/dataverse/export/ddi/dataset-create-new-all-ddi-fields.json

```diff
@@ -51,9 +51,9 @@
           },
           {
             "typeName": "alternativeTitle",
-            "multiple": false,
+            "multiple": true,
             "typeClass": "primitive",
-            "value": "Alternative Title"
+            "value": ["Alternative Title"]
```

Comment:
It looks like API users will encounter backward incompatibility. Is that right? If so, can this please be added to the release note snippet?

Reply:
Yes, repeatable altTitl is valid DDI. See https://ddialliance.org/Specification/DDI-Codebook/2.5/XMLSchema/field_level_documentation_files/schemas/codebook_xsd/elements/titlStmt.html#a19 It has maxOccurs="unbounded"

I updated dataset-finch xml and json and added multiple alternative titles such that validity test can test it.

I also added comment about incompatibility into release note.

Reply:
Could you please add a second value here, just like in dataset-finch1.json - 
```
["Alternative Title1", "Alternative Title2"]
```
and that will need a corresponding change in `src/test/java/edu/harvard/iq/dataverse/export/ddi/exportfull.xml` that DdiExportUtilTest compares against. 
(I'm putting together a list of all the changes that need to be made when fields are made repeatable, for other developers)
Thank you!

Reply:
@landreev I added ``["Alternative Title1", "Alternative Title2"]`` into dataset-create-new-all-ddi-fields.json and added another title in corresponding exportfull.xml